src/database/setup.js

```diff
@@ -6,6 +6,11 @@ const createDatabases = async () => {
         await createDatabase('credentials');
         await createDatabase('trivia_games');
     } catch (error) {
+
+        if (error.code == 'ECONNREFUSED') {
+            console.error('Error: Failed to connect to the MySQL database. Please ensure that the MySQL server is running and accessible.');
```

Comment:
Perhaps delete 'Error: '

Reply:
Perhaps also provide the error itself

Reply:
> Perhaps delete 'Error: '

Affected file: src/database/setup.js
https://github.com/Syncora/Syncora-Backend/pull/12/commits/d9584972893e8121a3a0de4f30886e943b05cb29